### ISSUE TITLE
[Question] ARM64 support

### ISSUE THREAD

**Issue by Vonng** [polardb/PolarDB-for-PostgreSQL#542]
**Describe the problem**

有计划提供 ARM64 架构的软件包吗？

**Comment by mrdrivingduck** [comment]
@Vonng Yes. Technically, all of our packages are packaged inside corresponding Docker container, and our Docker image is multi-arch which supports aarch64. However, the packaging is done on GitHub Actions, we are waiting for GitHub to support [native aarch64 runners](https://github.blog/news-insights/product-news/arm64-on-github-actions-powering-faster-more-efficient-build-systems/).

**Comment by mrdrivingduck** [comment]
@Vonng Supported as GitHub has supported native aarch64 runner on 20250117. See [release](https://github.com/ApsaraDB/PolarDB-for-PostgreSQL/releases).
